pipelines/skylab/smartseq2_single_sample/SmartSeq2SingleSample.changelog.md

```diff
@@ -1,3 +1,8 @@
+# 5.1.4
+2021-09-02 (Date of Last Commit)
+
+* Updated picard task to accomodate Star-Solo based Optimus
```

Comment:
Same comment as the one on the SmartSeq2SingleNucleus.changelog.md entry

Reply:
updated.